Warnings of usage of depricated code within Arduin Cloud libraries

Hi,
I use Arduino Cloud with ESP8266 and compilatoin of the code reports a lot of warnings about usage of depricated functions. However, they all come from within the libraries itself, I do not use those call explicitly at all. See compiler output:
```
In file included from d:\Dropbox\Arduino\sketch\ArduinoCloud\ArduinoCloud.ino:1:
d:\Dropbox\Arduino\sketch\ArduinoCloud\ArduinoCloud.ino: In function 'void setup()':
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src/ArduinoIoTCloud.h:113:68: warning: 'void ArduinoIoTCloudClass::addPropertyReal(bool&, String, permissionType, long int, void (*)(), float, void (*)(Property&))' is deprecated: Use addProperty(property, Permission::ReadWrite) instead. [-Wdeprecated-declarations]
113 |     #define addProperty( v, ...) addPropertyReal(v, #v, __VA_ARGS__)
|                                                                    ^
d:\Dropbox\Arduino\sketch\ArduinoCloud\ArduinoCloud.ino:29:16: note: in expansion of macro 'addProperty'
29 |   ArduinoCloud.addProperty(btnState, READWRITE, ON_CHANGE, onBtnStateChange);
|                ^~~~~~~~~~~
In file included from d:\Dropbox\Arduino\sketch\ArduinoCloud\ArduinoCloud.ino:1:
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src/ArduinoIoTCloud.h:120:10: note: declared here
120 |     void addPropertyReal(bool& property, String name, permissionType permission_type = READWRITE, long seconds = ON_CHANGE, void(*fn)(void) = NULL, float minDelta = 0.0f, void(*synFn)(Property & property) = CLOUD_WINS) __attribute__((deprecated("Use addProperty(property, Permission::ReadWrite) instead.")));
|          ^~~~~~~~~~~~~~~
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\ArduinoIoTCloud.cpp: In member function 'void ArduinoIoTCloudClass::addPropertyReal(bool&, String, permissionType, long int, void (*)(), float, void (*)(Property&))':
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\ArduinoIoTCloud.cpp:129:84: warning: 'void ArduinoIoTCloudClass::addPropertyReal(bool&, String, int, permissionType, long int, void (*)(), float, void (*)(Property&))' is deprecated: Use addProperty(property, Permission::ReadWrite) instead. [-Wdeprecated-declarations]
129 |   addPropertyReal(property, name, -1, permission_type, seconds, fn, minDelta, synFn);
|                                                                                    ^
In file included from D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\ArduinoIoTCloud.cpp:22:
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src/ArduinoIoTCloud.h:140:10: note: declared here
140 |     void addPropertyReal(bool& property, String name, int tag, permissionType permission_type = READWRITE, long seconds = ON_CHANGE, void(*fn)(void) = NULL, float minDelta = 0.0f, void(*synFn)(Property & property) = CLOUD_WINS) __attribute__((deprecated("Use addProperty(property, Permission::ReadWrite) instead.")));
|          ^~~~~~~~~~~~~~~
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\ArduinoIoTCloud.cpp: In member function 'void ArduinoIoTCloudClass::addPropertyReal(float&, String, permissionType, long int, void (*)(), float, void (*)(Property&))':
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\ArduinoIoTCloud.cpp:133:84: warning: 'void ArduinoIoTCloudClass::addPropertyReal(float&, String, int, permissionType, long int, void (*)(), float, void (*)(Property&))' is deprecated: Use addProperty(property, Permission::ReadWrite) instead. [-Wdeprecated-declarations]
133 |   addPropertyReal(property, name, -1, permission_type, seconds, fn, minDelta, synFn);
|                                                                                    ^
In file included from D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\ArduinoIoTCloud.cpp:22:
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src/ArduinoIoTCloud.h:141:10: note: declared here
141 |     void addPropertyReal(float& property, String name, int tag, permissionType permission_type = READWRITE, long seconds = ON_CHANGE, void(*fn)(void) = NULL, float minDelta = 0.0f, void(*synFn)(Property & property) = CLOUD_WINS) __attribute__((deprecated("Use addProperty(property, Permission::ReadWrite) instead.")));
|          ^~~~~~~~~~~~~~~
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\ArduinoIoTCloud.cpp: In member function 'void ArduinoIoTCloudClass::addPropertyReal(int&, String, permissionType, long int, void (*)(), float, void (*)(Property&))':
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\ArduinoIoTCloud.cpp:137:84: warning: 'void ArduinoIoTCloudClass::addPropertyReal(int&, String, int, permissionType, long int, void (*)(), float, void (*)(Property&))' is deprecated: Use addProperty(property, Permission::ReadWrite) instead. [-Wdeprecated-declarations]
137 |   addPropertyReal(property, name, -1, permission_type, seconds, fn, minDelta, synFn);
|                                                                                    ^
In file included from D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\ArduinoIoTCloud.cpp:22:
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src/ArduinoIoTCloud.h:142:10: note: declared here
142 |     void addPropertyReal(int& property, String name, int tag, permissionType permission_type = READWRITE, long seconds = ON_CHANGE, void(*fn)(void) = NULL, float minDelta = 0.0, void(*synFn)(Property & property) = CLOUD_WINS) __attribute__((deprecated("Use addProperty(property, Permission::ReadWrite) instead.")));
|          ^~~~~~~~~~~~~~~
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\ArduinoIoTCloud.cpp: In member function 'void ArduinoIoTCloudClass::addPropertyReal(unsigned int&, String, permissionType, long int, void (*)(), float, void (*)(Property&))':
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\ArduinoIoTCloud.cpp:141:84: warning: 'void ArduinoIoTCloudClass::addPropertyReal(unsigned int&, String, int, permissionType, long int, void (*)(), float, void (*)(Property&))' is deprecated: Use addProperty(property, Permission::ReadWrite) instead. [-Wdeprecated-declarations]
141 |   addPropertyReal(property, name, -1, permission_type, seconds, fn, minDelta, synFn);
|                                                                                    ^
In file included from D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\ArduinoIoTCloud.cpp:22:
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src/ArduinoIoTCloud.h:143:10: note: declared here
143 |     void addPropertyReal(unsigned int& property, String name, int tag, permissionType permission_type = READWRITE, long seconds = ON_CHANGE, void(*fn)(void) = NULL, float minDelta = 0.0, void(*synFn)(Property & property) = CLOUD_WINS) __attribute__((deprecated("Use addProperty(property, Permission::ReadWrite) instead.")));
|          ^~~~~~~~~~~~~~~
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\ArduinoIoTCloud.cpp: In member function 'void ArduinoIoTCloudClass::addPropertyReal(String&, String, permissionType, long int, void (*)(), float, void (*)(Property&))':
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\ArduinoIoTCloud.cpp:145:84: warning: 'void ArduinoIoTCloudClass::addPropertyReal(String&, String, int, permissionType, long int, void (*)(), float, void (*)(Property&))' is deprecated: Use addProperty(property, Permission::ReadWrite) instead. [-Wdeprecated-declarations]
145 |   addPropertyReal(property, name, -1, permission_type, seconds, fn, minDelta, synFn);
|                                                                                    ^
In file included from D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\ArduinoIoTCloud.cpp:22:
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src/ArduinoIoTCloud.h:144:10: note: declared here
144 |     void addPropertyReal(String& property, String name, int tag, permissionType permission_type = READWRITE, long seconds = ON_CHANGE, void(*fn)(void) = NULL, float minDelta = 0.0f, void(*synFn)(Property & property) = CLOUD_WINS) __attribute__((deprecated("Use addProperty(property, Permission::ReadWrite) instead.")));
|          ^~~~~~~~~~~~~~~
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\ArduinoIoTCloud.cpp: In member function 'void ArduinoIoTCloudClass::addPropertyReal(Property&, String, permissionType, long int, void (*)(), float, void (*)(Property&))':
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\ArduinoIoTCloud.cpp:149:84: warning: 'void ArduinoIoTCloudClass::addPropertyReal(Property&, String, int, permissionType, long int, void (*)(), float, void (*)(Property&))' is deprecated: Use addProperty(property, Permission::ReadWrite) instead. [-Wdeprecated-declarations]
149 |   addPropertyReal(property, name, -1, permission_type, seconds, fn, minDelta, synFn);
|                                                                                    ^
In file included from D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\ArduinoIoTCloud.cpp:22:
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src/ArduinoIoTCloud.h:139:10: note: declared here
139 |     void addPropertyReal(Property& property, String name, int tag, permissionType permission_type = READWRITE, long seconds = ON_CHANGE, void(*fn)(void) = NULL, float minDelta = 0.0f, void(*synFn)(Property & property) = CLOUD_WINS) __attribute__((deprecated("Use addProperty(property, Permission::ReadWrite) instead.")));
|          ^~~~~~~~~~~~~~~
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\ArduinoIoTCloud.cpp: In member function 'void ArduinoIoTCloudClass::addPropertyReal(bool&, String, int, permissionType, long int, void (*)(), float, void (*)(Property&))':
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\ArduinoIoTCloud.cpp:156:79: warning: 'void ArduinoIoTCloudClass::addPropertyReal(Property&, String, int, permissionType, long int, void (*)(), float, void (*)(Property&))' is deprecated: Use addProperty(property, Permission::ReadWrite) instead. [-Wdeprecated-declarations]
156 |   addPropertyReal(*p, name, tag, permission_type, seconds, fn, minDelta, synFn);
|                                                                               ^
In file included from D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\ArduinoIoTCloud.cpp:22:
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src/ArduinoIoTCloud.h:139:10: note: declared here
139 |     void addPropertyReal(Property& property, String name, int tag, permissionType permission_type = READWRITE, long seconds = ON_CHANGE, void(*fn)(void) = NULL, float minDelta = 0.0f, void(*synFn)(Property & property) = CLOUD_WINS) __attribute__((deprecated("Use addProperty(property, Permission::ReadWrite) instead.")));
|          ^~~~~~~~~~~~~~~
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\ArduinoIoTCloud.cpp: In member function 'void ArduinoIoTCloudClass::addPropertyReal(float&, String, int, permissionType, long int, void (*)(), float, void (*)(Property&))':
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\ArduinoIoTCloud.cpp:161:79: warning: 'void ArduinoIoTCloudClass::addPropertyReal(Property&, String, int, permissionType, long int, void (*)(), float, void (*)(Property&))' is deprecated: Use addProperty(property, Permission::ReadWrite) instead. [-Wdeprecated-declarations]
161 |   addPropertyReal(*p, name, tag, permission_type, seconds, fn, minDelta, synFn);
|                                                                               ^
In file included from D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\ArduinoIoTCloud.cpp:22:
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src/ArduinoIoTCloud.h:139:10: note: declared here
139 |     void addPropertyReal(Property& property, String name, int tag, permissionType permission_type = READWRITE, long seconds = ON_CHANGE, void(*fn)(void) = NULL, float minDelta = 0.0f, void(*synFn)(Property & property) = CLOUD_WINS) __attribute__((deprecated("Use addProperty(property, Permission::ReadWrite) instead.")));
|          ^~~~~~~~~~~~~~~
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\ArduinoIoTCloud.cpp: In member function 'void ArduinoIoTCloudClass::addPropertyReal(int&, String, int, permissionType, long int, void (*)(), float, void (*)(Property&))':
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\ArduinoIoTCloud.cpp:166:79: warning: 'void ArduinoIoTCloudClass::addPropertyReal(Property&, String, int, permissionType, long int, void (*)(), float, void (*)(Property&))' is deprecated: Use addProperty(property, Permission::ReadWrite) instead. [-Wdeprecated-declarations]
166 |   addPropertyReal(*p, name, tag, permission_type, seconds, fn, minDelta, synFn);
|                                                                               ^
In file included from D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\ArduinoIoTCloud.cpp:22:
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src/ArduinoIoTCloud.h:139:10: note: declared here
139 |     void addPropertyReal(Property& property, String name, int tag, permissionType permission_type = READWRITE, long seconds = ON_CHANGE, void(*fn)(void) = NULL, float minDelta = 0.0f, void(*synFn)(Property & property) = CLOUD_WINS) __attribute__((deprecated("Use addProperty(property, Permission::ReadWrite) instead.")));
|          ^~~~~~~~~~~~~~~
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\ArduinoIoTCloud.cpp: In member function 'void ArduinoIoTCloudClass::addPropertyReal(unsigned int&, String, int, permissionType, long int, void (*)(), float, void (*)(Property&))':
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\ArduinoIoTCloud.cpp:171:79: warning: 'void ArduinoIoTCloudClass::addPropertyReal(Property&, String, int, permissionType, long int, void (*)(), float, void (*)(Property&))' is deprecated: Use addProperty(property, Permission::ReadWrite) instead. [-Wdeprecated-declarations]
171 |   addPropertyReal(*p, name, tag, permission_type, seconds, fn, minDelta, synFn);
|                                                                               ^
In file included from D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\ArduinoIoTCloud.cpp:22:
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src/ArduinoIoTCloud.h:139:10: note: declared here
139 |     void addPropertyReal(Property& property, String name, int tag, permissionType permission_type = READWRITE, long seconds = ON_CHANGE, void(*fn)(void) = NULL, float minDelta = 0.0f, void(*synFn)(Property & property) = CLOUD_WINS) __attribute__((deprecated("Use addProperty(property, Permission::ReadWrite) instead.")));
|          ^~~~~~~~~~~~~~~
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\ArduinoIoTCloud.cpp: In member function 'void ArduinoIoTCloudClass::addPropertyReal(String&, String, int, permissionType, long int, void (*)(), float, void (*)(Property&))':
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\ArduinoIoTCloud.cpp:176:79: warning: 'void ArduinoIoTCloudClass::addPropertyReal(Property&, String, int, permissionType, long int, void (*)(), float, void (*)(Property&))' is deprecated: Use addProperty(property, Permission::ReadWrite) instead. [-Wdeprecated-declarations]
176 |   addPropertyReal(*p, name, tag, permission_type, seconds, fn, minDelta, synFn);
|                                                                               ^
In file included from D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\ArduinoIoTCloud.cpp:22:
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src/ArduinoIoTCloud.h:139:10: note: declared here
139 |     void addPropertyReal(Property& property, String name, int tag, permissionType permission_type = READWRITE, long seconds = ON_CHANGE, void(*fn)(void) = NULL, float minDelta = 0.0f, void(*synFn)(Property & property) = CLOUD_WINS) __attribute__((deprecated("Use addProperty(property, Permission::ReadWrite) instead.")));
|          ^~~~~~~~~~~~~~~
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\utility\time\TimeService.cpp: In member function 'bool TimeServiceClass::connected()':
D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\utility\time\TimeService.cpp:284:32: warning: 'NetworkConnectionState ConnectionHandler::getStatus()' is deprecated [-Wdeprecated-declarations]
284 |     return _con_hdl->getStatus() == NetworkConnectionState::CONNECTED;
|                                ^
In file included from D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\utility\time\TimeService.h:26,
from D:\Dropbox\Arduino\sketch\libraries\arduino_911903\src\utility\time\TimeService.cpp:25:
D:\Dropbox\Arduino\sketch\libraries\arduino_797336\src/Arduino_ConnectionHandler.h:252:28: note: declared here
252 |     NetworkConnectionState getStatus() __attribute__((deprecated)) {
|                            ^~~~~~~~~
```

--- a/src/ArduinoIoTCloud.cpp
+++ b/src/ArduinoIoTCloud.cpp
@@ -126,56 +126,66 @@ Property& ArduinoIoTCloudClass::addPropertyReal(Property& property, String name,
 /* The following methods are deprecated but still used for non-LoRa boards */
 void ArduinoIoTCloudClass::addPropertyReal(bool& property, String name, permissionType permission_type, long seconds, void(*fn)(void), float minDelta, void(*synFn)(Property & property))
 {
-  addPropertyReal(property, name, -1, permission_type, seconds, fn, minDelta, synFn);
+  Property* p = new CloudWrapperBool(property);
+  addPropertyRealInternal(*p, name, -1, permission_type, seconds, fn, minDelta, synFn);
 }
 void ArduinoIoTCloudClass::addPropertyReal(float& property, String name, permissionType permission_type, long seconds, void(*fn)(void), float minDelta, void(*synFn)(Property & property))
 {
-  addPropertyReal(property, name, -1, permission_type, seconds, fn, minDelta, synFn);
+  Property* p = new CloudWrapperFloat(property);
+  addPropertyRealInternal(*p, name, -1, permission_type, seconds, fn, minDelta, synFn);
 }
 void ArduinoIoTCloudClass::addPropertyReal(int& property, String name, permissionType permission_type, long seconds, void(*fn)(void), float minDelta, void(*synFn)(Property & property))
 {
-  addPropertyReal(property, name, -1, permission_type, seconds, fn, minDelta, synFn);
+  Property* p = new CloudWrapperInt(property);
+  addPropertyRealInternal(*p, name, -1, permission_type, seconds, fn, minDelta, synFn);
 }
 void ArduinoIoTCloudClass::addPropertyReal(unsigned int& property, String name, permissionType permission_type, long seconds, void(*fn)(void), float minDelta, void(*synFn)(Property & property))
 {
-  addPropertyReal(property, name, -1, permission_type, seconds, fn, minDelta, synFn);
+  Property* p = new CloudWrapperUnsignedInt(property);
+  addPropertyRealInternal(*p, name, -1, permission_type, seconds, fn, minDelta, synFn);
 }
 void ArduinoIoTCloudClass::addPropertyReal(String& property, String name, permissionType permission_type, long seconds, void(*fn)(void), float minDelta, void(*synFn)(Property & property))
 {
-  addPropertyReal(property, name, -1, permission_type, seconds, fn, minDelta, synFn);
+  Property* p = new CloudWrapperString(property);
+  addPropertyRealInternal(*p, name, -1, permission_type, seconds, fn, minDelta, synFn);
 }
 void ArduinoIoTCloudClass::addPropertyReal(Property& property, String name, permissionType permission_type, long seconds, void(*fn)(void), float minDelta, void(*synFn)(Property & property))
 {
-  addPropertyReal(property, name, -1, permission_type, seconds, fn, minDelta, synFn);
+  addPropertyRealInternal(property, name, -1, permission_type, seconds, fn, minDelta, synFn);
 }
 
 /* The following methods are deprecated but still used for both LoRa and non-LoRa boards */
 void ArduinoIoTCloudClass::addPropertyReal(bool& property, String name, int tag, permissionType permission_type, long seconds, void(*fn)(void), float minDelta, void(*synFn)(Property & property))
 {
   Property* p = new CloudWrapperBool(property);
-  addPropertyReal(*p, name, tag, permission_type, seconds, fn, minDelta, synFn);
+  addPropertyRealInternal(*p, name, tag, permission_type, seconds, fn, minDelta, synFn);
 }
 void ArduinoIoTCloudClass::addPropertyReal(float& property, String name, int tag, permissionType permission_type, long seconds, void(*fn)(void), float minDelta, void(*synFn)(Property & property))
 {
   Property* p = new CloudWrapperFloat(property);
-  addPropertyReal(*p, name, tag, permission_type, seconds, fn, minDelta, synFn);
+  addPropertyRealInternal(*p, name, tag, permission_type, seconds, fn, minDelta, synFn);
 }
 void ArduinoIoTCloudClass::addPropertyReal(int& property, String name, int tag, permissionType permission_type, long seconds, void(*fn)(void), float minDelta, void(*synFn)(Property & property))
 {
   Property* p = new CloudWrapperInt(property);
-  addPropertyReal(*p, name, tag, permission_type, seconds, fn, minDelta, synFn);
+  addPropertyRealInternal(*p, name, tag, permission_type, seconds, fn, minDelta, synFn);
 }
 void ArduinoIoTCloudClass::addPropertyReal(unsigned int& property, String name, int tag, permissionType permission_type, long seconds, void(*fn)(void), float minDelta, void(*synFn)(Property & property))
 {
   Property* p = new CloudWrapperUnsignedInt(property);
-  addPropertyReal(*p, name, tag, permission_type, seconds, fn, minDelta, synFn);
+  addPropertyRealInternal(*p, name, tag, permission_type, seconds, fn, minDelta, synFn);
 }
 void ArduinoIoTCloudClass::addPropertyReal(String& property, String name, int tag, permissionType permission_type, long seconds, void(*fn)(void), float minDelta, void(*synFn)(Property & property))
 {
   Property* p = new CloudWrapperString(property);
-  addPropertyReal(*p, name, tag, permission_type, seconds, fn, minDelta, synFn);
+  addPropertyRealInternal(*p, name, tag, permission_type, seconds, fn, minDelta, synFn);
 }
 void ArduinoIoTCloudClass::addPropertyReal(Property& property, String name, int tag, permissionType permission_type, long seconds, void(*fn)(void), float minDelta, void(*synFn)(Property & property))
+{
+  addPropertyRealInternal(property, name, tag, permission_type, seconds, fn, minDelta, synFn);
+}
+
+void ArduinoIoTCloudClass::addPropertyRealInternal(Property& property, String name, int tag, permissionType permission_type, long seconds, void(*fn)(void), float minDelta, void(*synFn)(Property & property))
 {
   Permission permission = Permission::ReadWrite;
   if (permission_type == READ) {

--- a/src/ArduinoIoTCloud.h
+++ b/src/ArduinoIoTCloud.h
@@ -166,6 +166,8 @@ class ArduinoIoTCloudClass
 
   private:
 
+    void addPropertyRealInternal(Property& property, String name, int tag, permissionType permission_type = READWRITE, long seconds = ON_CHANGE, void(*fn)(void) = NULL, float minDelta = 0.0f, void(*synFn)(Property & property) = CLOUD_WINS);
+
     String _device_id;
     OnCloudEventCallback _cloud_event_callback[3];
     bool _thing_id_outdated;

